### DISCUSSION TITLE
Trouble with Verifying School email for Student Package.

### DISCUSSION THREAD

**Discussion by Enigmatak** [community/community#120800] [GitHub Education]
### Hi everyone,

Hello! I am very new to using GitHub so I apologize if my terminology and is incorrect. 

My problem: 
I am trying to sign up for the student package that GitHub offers. I am currently enrolled in a community college, which is easily findable when searching what school to add in the sign up process. When it comes to verifying my school email, I run into an issue. Every time I press “send verify school email”, nothing happens. However, when I remove the school email from my account, I *do* get mail sent to the address I tried to verify. It’s so frustrating. I have checked spams, and snooped around the settings related to the account (it is an outlook email address). I have my student id and plenty of other verification I am willing to give, but I am not sure what to do otherwise. Any tips or resources would be appreciated! Thanks.

edit: spelling

**Comment by samus-aran** [comment]
Hi @Enigmatak 

May sound like your school email is either blocking external mail or GitHub needs to be whitelisted. Confirm with your school's IT team first.
